src/whereInt.h

```diff
@@ -19,7 +19,7 @@
 ** Trace output macros
 */
 #if defined(SQLITE_TEST) || defined(SQLITE_DEBUG)
-/***/ int sqlite3WhereTrace;
+/***/ extern int sqlite3WhereTrace;
```

Comment:
Even if we think the rest of this is somewhat CHERI-specific, should this go upstream?

Reply:
I believe this is an arbitrary old version and I assume it might already be fixed upstream since it's the default for GCC/Clang 10

Reply:
It's a deliberately old version to have vulnerabilities.  We don't require nearly so many patches on the newer version.